awscli/clidocs.py

```diff
@@ -159,6 +159,7 @@ def doc_option(self, arg_name, help_command, **kwargs):
             name = '``%s``' % argument.cli_name
         doc.write('%s (%s)\n' % (name, argument.cli_type_name))
         doc.style.indent()
+        self._document_enums(argument, doc)
```

Comment:
I feel that it may look better if the enum values are documented after the doc string. Thoughts?